README.md

```diff
@@ -3,22 +3,17 @@
 
 ## Intro
 
-PnP is a set of technologies designed to make it easier to build IoT Solutions based on Azure IoT Hub. To address the needs for device and cloud developers the tutorial must target both.
-
-Devices connected to IoT Hub can de described with DTDL, in terms of:
-
-- Properties (Available trough Twins)
-- Commands (Using the Hub Direct Methods feature)
-- Telemetry (processed through EventHub)
+This bug bash explores the new IoT Hub features to support the Plug and Play May 2020 release. Using a  DTDL v2 Model to describe a device with one  sensor interface that sends telemetry, accepts commands and reports properties. The sample is based on a device simulator connected to IoT Hub with a client SDK and can be managed using the service SDKs  to read telemetry and interact with properties and commands.
 
 Review the docs in the private staging environment [IoT Plug and Play Preview Documentation](https://review.docs.microsoft.com/azure/iot-pnp/?branch=pr-en-us-114283)
 
+[DTDL v2 Spec](https://microsoft.sharepoint.com/:w:/t/Azure_IoT/IoTPlat/EaotICYXiQVOh_xnBggJwqMBMKGoOyMzV1qFYsOdcyvOVw?e=VyG0QN) (internal)
```

Comment:
I recommend calling out this is for reference only.  It's a long spec and I don't think a bugbasher could finish all of this in the time alloted.